config/defaults.conf

```diff
@@ -250,14 +290,21 @@ dropout = 0.2  // Dropout rate.
 dropout_embs = ${dropout}  // Dropout rate for embeddings, same as above by default.
                            // NB: This only applies to trained char embs, not including ELMo.
 
+onlstm_chunk_size = 10  // Chunk downsizing factor for ON-LSTM master gate, dimensions
+                        // of master gate: D/C where C is the chunk downsizing factor.
+onlstm_dropconnect = 0.5  // Linear dropout between ON-LSTM layers.
+onlstm_dropouti  = 0.3  // Locked Dropout on input embeddings.
+onlstm_dropouth  = 0.3  // Locked Dropout between ON-LSTM layers.
+onlstm_tying = 0  // Language Modeling tying of weights.
+
 
 // Task-specific Options //
-// These are _usually_ overridden for specific tasks, and are explicitly overridden in this file for many tasks, but defaults
-// are set here.
+
+// These are _usually_ overridden for specific tasks, and are explicitly overridden in this file for many tasks, but defaults are set here.
 
 // Model
-pool_type = max // The type of pooling to aggregate sequences of vectors into a single vector.
-                // Options: first, last, max, mean.
+pool_type = max  // The type of pooling to aggregate sequences of vectors into a single vector.
```

Comment:
I'm pretty sure this isn't doing anything right now and it's an issue to let it actually do things.